### PR TITLE
Expliciting SQL Server connector limitation for materialized views

### PR DESCRIPTION
**Pull request by danielpetisme**
A proposition of doc for https://issues.redhat.com/browse/DBZ-4330

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/sqlserver.adoc [hunk 1]
```diff
@@ -152,6 +152,14 @@ After a restart, the connector resumes processing from the last offset (commit a
 
 The connector is able to detect whether CDC is enabled or disabled for included source tables and adjust its behavior.
 
+// Type: concept
+// ModuleID: limitations-sql-server-connector
+// Title: Limitations of {prodname} SQL Server connector
+=== Limitations
+
+SQL Server specifically requires the base object to be a table in order to create a change capture instance.
+As consequence, capturing changes from indexed views (aka. materialized views) is not supported by SQL Sever and hence {prodname} SQL Server connector.
```
Comment (gunnarmorling) [inline documentation/modules/ROOT/pages/connectors/sqlserver.adoc:161]:
```suggestion
As consequence, capturing changes from indexed views (aka. materialized views) is not supported by SQL Server and hence {prodname} SQL Server connector.
```

**Review by gunnarmorling — APPROVED** [review]
Thanks a lot, @danielpetisme! This LGTM.

Is there any reference in SQL Server docs we could link to?

**Comment by gunnarmorling** [comment]
Fixed the typo and applied. Thanks a lot, @danielpetisme!

**Comment by danielpetisme** [comment]
Unfortunately, I can't find an official mention of the limitation on MS doc.
It's a kind of implicit thing since only tables are mentioned...

**Pull request merged**